packages/unimodules-permissions-interface/ios/UMPermissionsInterface/UMPermissionsMethodsDelegate.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+#import <UMPermissionsInterface/UMPermissionsInterface.h>
+
+@interface UMPermissionsMethodsDelegate : NSObject
+
++ (void)getPermissionWithPermissionsManager:(id<UMPermissionsInterface>)permissionsManager
+                              withRequester:(Class)requesterClass
+                                 withResult:(UMPromiseResolveBlock)resolve
```

Comment:
same comment for both of these as well -- use `resolver:resolve` and `rejecter:reject`